mmengine/runner/runner.py

```diff
@@ -355,6 +355,7 @@ def __init__(
         # `set_randomness`` method
         self._randomness_cfg = randomness
         self.set_randomness(**randomness)
+        self._randomness_cfg['seed'] = self._seed
```

Comment:
It's not recommended to modify `self._randomness_cfg`, just update it when logging the environment. https://github.com/open-mmlab/mmengine/blob/ec2e00ae9060572de96759ef141254d352134d39/mmengine/runner/runner.py#L2308

```python
runtime_env.update(self._randomness_cfg)
runtime_env['seed'] = self._seed
```